nemo_skills/pipeline/summarize_results.py

```diff
@@ -105,15 +105,50 @@ def summarize_results(
         results_dir = Path(temp_dir) / Path(results_dir).name
 
     # running compute_metrics.py to get greedy, majority and pass @k results for all benchmarks available
-    # Check if there is an eval-results dir inside the results_dir
-    eval_results_dir = Path(results_dir) / 'eval-results'
-    if eval_results_dir.exists() and eval_results_dir.is_dir():
-        results_dir = eval_results_dir
-    benchmarks_paths = [path for path in glob.glob(f'{results_dir}/*') if '-logs' not in os.path.basename(path)]
+
+    # Check for all possible directory structures
+    # 1. {results_dir}/eval-results/{benchmark}/output*jsonl
+    # 2. {results_dir}/{benchmark}/output*jsonl
+    # 3. {results_dir}/output*jsonl
+
+    # List to store all valid benchmarks results paths
+    benchmarks_paths = []
+
+    # Check for Option 3 - Root directory corresponds to a benchmark
+    if Path(results_dir).is_dir() and len(glob.glob(f'{results_dir}/output*jsonl')) > 0:
+        benchmarks_paths = [results_dir]
+    else:
+        cand_results_dir = Path(results_dir) / 'eval-results'
+        # Check for Option 1
+        if cand_results_dir.exists() and cand_results_dir.is_dir():
+            results_dir = cand_results_dir
+        elif Path(results_dir).name == 'eval-results':
```

Comment:
Let's remove this check - it doesn't have to be named "eval-results" as you can run this on the output of the judge pipeline or generate pipeline

Reply:
How about replacing this with:
`elif len(glob.glob(f'{results_dir}/*/output*jsonl')) > 0:`

This way, you can supply `/.../eval-results` and `/.../eval-results-judged` as direct inputs.